### PR TITLE
feat: update schema with information about artifact file

### PR DESCRIPTION
**Pull request by Smeds**

### CODE REVIEW

File: workflow/schemas/config.schema.yaml [hunk 1]
```diff
@@ -147,6 +147,17 @@ properties:
...
+  reference:
+    type: object
+    description: reference used by alignment
+    properties:
+      artifacts:
+        type: string
+        description: artifact file
+      background:
+        type: string
+        description: artifact file
```
Comment (jonca79) [inline workflow/schemas/config.schema.yaml:159]: background file

Comment (Smeds) [reply workflow/schemas/config.schema.yaml]: Changed

File: workflow/schemas/config.schema.yaml [hunk 2]
```diff
@@ -147,6 +147,17 @@ properties:
         type: string
         description: name or path to docker/singularity container
 
+  reference:
+    type: object
+    description: reference used by alignment
```
Comment (marrip) [inline workflow/schemas/config.schema.yaml:152]: It doesn't make sense to say `reference used for alignment` when the subcategories are artifacts and background. Maybe simply `collection of reference files` or something similar

File: workflow/schemas/config.schema.yaml [hunk 3]
```diff
@@ -147,6 +147,17 @@ properties:
...
+    properties:
+      artifacts:
+        type: string
+        description: artifact file
```
Comment (marrip) [inline workflow/schemas/config.schema.yaml:156]: This description does not explain what kind of file it is and that it is a path. I think we should be specific, same for background

File: workflow/schemas/config.schema.yaml (hunk 2 above)
Comment (Smeds) [inline workflow/schemas/config.schema.yaml:152]: `description: collection of reference files`

Comment (Smeds) [inline workflow/schemas/config.schema.yaml:152]:
```suggestion
    description: collection of reference files
```

File: workflow/schemas/config.schema.yaml (hunk 3 above)
Comment (Smeds) [reply workflow/schemas/config.schema.yaml:156]: @jonca79  could you give a suggestion for an appropriate text

Comment (jonca79) [reply workflow/schemas/config.schema.yaml:156]: description: path to artifact file containing caller specific variant observation counts
description: path to background file containing median and sd values for non-reference bases per position

Comment (Smeds) [reply workflow/schemas/config.schema.yaml]: thanks

**Review by jonca79 — APPROVED** [review]

**Pull request merged**